lib/index.js

```diff
@@ -141,11 +141,11 @@ const open = (_args, opts = {}, extra = {}) => {
     if (platform === 'win32') {
       // spawnWithShell does not do the additional os.release() check, so we
       // have to force the shell here to make sure we treat WSL as windows.
-      options.shell = process.env.ComSpec
+      const shellCommand = process.env.ComSpec || 'cmd.exe'
       // also, the start command accepts a title so to make sure that we don't
       // accidentally interpret the first arg as the title, we stick an empty
       // string immediately after the start command
-      command = 'start ""'
+      command = `${shellCommand} /c start ""`
```

Comment:
removing `options.shell` is likely why this had to change. it's not the right solution, you need to keep `options.shell` set and let the rest of the code build the `/c` part.